v3/src/pages/docs/core-concepts/activity.md

```diff
@@ -116,6 +116,24 @@ The following is an example of annotating an activity with metadata:
     Category = "Demo",
     Description = "A simple activity that writes \"Hello World!\" to the console."
 )]
+```
+ Variables inputs Metadata can also be annotated using the `InputAttribute` attribute.
+ Simmilar to above, this allows the Designer to display "friendly names" and descriptions of an input.
```

Comment:
How about:

"Activity inputs can be annotated using the `InputAttribute` attribute.
 Similar to above, this allows the designer to display "friendly names" and descriptions of an input."